lib/bundles/inspec-compliance/api.rb

```diff
@@ -27,17 +27,22 @@ def self.profiles(config)
       # TODO, api should not be dependent on .supported?
       response = Compliance::HTTP.get(url, config['token'], config['insecure'], !config.supported?(:oidc))
       data = response.body
-      if data != ''
+      response_code = response.code
+      case response_code
+      when '200'
         profiles = JSON.parse(data)
         # iterate over profiles
         profiles.map do |owner, ps|
           ps.keys.map do |name|
             { org: owner, name: name }
           end
         end.flatten
-      else
+      when '401'
         puts '401 Unauthorized. Please check your token.'
         []
+      else
+        puts response_code
```

Comment:
We should add a message that indicates an error. I assume most users do not know http codes by heart ;-)